From 591ec24748525171d5647d2a35cd1da1445e7a0b Mon Sep 17 00:00:00 2001
From: James Haggerty <james.haggerty@morsemicro.com>
Date: Fri, 26 Jul 2024 09:55:03 +1000
Subject: [PATCH] Hack out channel 50 (and typo for 51)

If we use these channels on a board with those channels disabled,
the DPP presence announcement fails.

This is rubbish: see SW-12450
---
 wpa_supplicant/dpp_supplicant.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/wpa_supplicant/dpp_supplicant.c b/wpa_supplicant/dpp_supplicant.c
index 09f8ceb07..46e4add95 100644
--- a/wpa_supplicant/dpp_supplicant.c
+++ b/wpa_supplicant/dpp_supplicant.c
@@ -5199,12 +5199,12 @@ static int * wpas_dpp_presence_ann_channels(struct wpa_supplicant *wpa_s,
 			static const int s1g_chirp_channels[] = {
 				/* 2MHz channels */
 				5190, 5230, 5270, 5310, 5510, 5550,
-				5630, 5670, 5755, 5795, 5835, 5875,
+				5630, 5670, 5755, 5795, 5835,
 				/* 1MHz channels */
 				5660, 5680, 5180, 5200, 5240, 5260,
 				5280, 5300, 5320, 5500, 5520, 5540,
 				5560, 5580, 5600, 5620, 5640, 5765,
-				5785, 5805, 5825, 5845, 5865, 5855
+				5785, 5805, 5825, 5845, 5865
 			};
 
 			if (chan->flag & (HOSTAPD_CHAN_DISABLED | HOSTAPD_CHAN_RADAR))
-- 
2.25.1

